src/tokens/TokenImport.test.tsx

```diff
@@ -1,14 +1,50 @@
-import { fireEvent, render } from '@testing-library/react-native'
+import { fireEvent, render, waitFor } from '@testing-library/react-native'
 import React from 'react'
 import { Provider } from 'react-redux'
+import erc20 from 'src/abis/IERC20'
 import { Screens } from 'src/navigator/Screens'
+import { publicClient } from 'src/viem'
+import networkConfig, { networkIdToNetwork } from 'src/web3/networkConfig'
 import { createMockStore, getMockStackScreenProps } from 'test/utils'
-import { mockCusdAddress } from 'test/values'
+import { mockCusdAddress, mockPoofAddress } from 'test/values'
+import {
+  CallExecutionError,
+  ContractFunctionExecutionError,
+  GetContractReturnType,
+  PublicClient,
+  TimeoutError,
+  WalletClient,
+  getContract,
+} from 'viem'
 import TokenImportScreen from './TokenImport'
+import mocked = jest.mocked
+
+jest.mock('viem', () => ({
+  ...jest.requireActual('viem'),
+  getContract: jest.fn(),
+}))
 
 const mockScreenProps = getMockStackScreenProps(Screens.TokenImport)
 
 describe('TokenImport', () => {
+  const mockBytecode = jest.fn()
+  const mockSymbol = jest.fn()
+
+  beforeEach(() => {
+    jest.resetAllMocks()
+    const client = publicClient[networkIdToNetwork[networkConfig.defaultNetworkId]]
```

Comment:
```suggestion
    const client = publicClient[Network.Celo]
```
maybe more straightforward?